build/common.sh

```diff
@@ -681,7 +682,8 @@ function kube::build::sync_to_container() {
   # As an exception, we need to sync generated files in staging/, because
   # they will not be re-generated by 'make'.
   kube::build::rsync \
-    --delete \
```

Comment:
No, there was no problem.. just being explicit (before is the default, but just calling it out).